packages/coachmark/src/Coachmark.ts

```diff
@@ -13,32 +13,346 @@ governing permissions and limitations under the License.
 import {
     CSSResultArray,
     html,
-    SpectrumElement,
+    nothing,
+    PropertyValues,
     TemplateResult,
 } from '@spectrum-web-components/base';
 import { property } from '@spectrum-web-components/base/src/decorators.js';
-
-import styles from './coachmark.css.js';
+import { when } from '@spectrum-web-components/base/src/directives.js';
+import { LikeAnchor } from '@spectrum-web-components/shared/src/like-anchor.js';
+import coachmarkStyles from './coachmark.css.js';
+import chevronStyles from '@spectrum-web-components/icon/src/spectrum-icon-chevron.css.js';
+import '@spectrum-web-components/icons-ui/icons/sp-icon-chevron200.js';
+import { Popover } from '@spectrum-web-components/popover';
+import { join } from 'lit/directives/join.js';
+import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { Placement } from '@spectrum-web-components/overlay';
+import { MediaType } from './CoachmarkItem.js';
+import type { CoachmarkItem } from './CoachmarkItem.js';
+import '@spectrum-web-components/asset/sp-asset.js';
+import '@spectrum-web-components/button/sp-button.js';
+import '@spectrum-web-components/quick-actions/sp-quick-actions.js';
 
 /**
  * @element sp-coachmark
+ * @slot cover-photo - This is the cover photo for Default and Quiet Coachmark
+ * @slot heading - HTML content to be listed as the heading
+ * @slot description - A description of the card
+ * @slot actions - an `sp-action-menu` element outlining actions to take on the represened object
  */
-export class Coachmark extends SpectrumElement {
+export class Coachmark extends LikeAnchor(Popover) {
```

Comment:
Using the LikeAnchor class could have unintended consequences. What was your intention in leveraging this vs Popover?

Reply:
`LikeAnchor class` is need for the below this.href in `sp-action-menu` to work. LikeAnchor class exports `this.href`
```
private stopPropagationOnHref(event: Event): void {
        if (this.href) {
            event.stopPropagation();
        }
    }
```